### DISCUSSION TITLE
How to restart ConverstionHandler after pressing /start command?

### DISCUSSION THREAD

**Discussion by yutanov** [python-telegram-bot/python-telegram-bot#2959] [Q&A] [answered]
I have a ConverstionHandler script. How to start this script again after user passed all the steps or if user pressed "/start"? After /start command user can see a welcome message, and then pressing OK button, an then starts ConversationHandler. But if user pressed /start command user can see welcome message again and OK button doesn't start ConversationHandler.

**Comment by harshil21** [comment, marked as answer]
Set the parameter [allow_reentry=True](https://python-telegram-bot.readthedocs.io/en/v14/telegram.ext.conversationhandler.html#telegram.ext.ConversationHandler.params.allow_reentry) in ConversationHandler
